website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Podman Desktop Release v0.10
+description: Podman Desktop v0.10 has been released!
+slug: podman-desktop-release-0.10
+authors: [deekay2310]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+---
+
+This blog post covers all that is new with the latest release of Podman Desktop v0.10!
+
+With Podman Desktop v0.10, you will be able to install or upgrade to the latest Podman Engine (v4.3.1). This is essential on getting started with Podman Desktop if you don’t have any other container engine configured.
```

Comment:
it's not true on Linux

the version is v0.10.0  (not v0.10)
I don't see why it's `essential`

And I would not make it as the highlight of the blog post

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/9ae20742cadf241c250c5faff504b3738d335f16